docs/README.md

```diff
@@ -0,0 +1,16 @@
+# Read the Docs - OpenTelemetry .NET
+
+[Read the Docs Link](https://opentelemetry-dotnet.readthedocs.io)
+
+## Build
```

Comment:
these are *not* for public users right? only for the repo maintainers to push new doc changes?

Reply:
maybe somewhere https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/build/RELEASING.md ?

Reply:
That is the public URL to view the documentation that will be built by readthedocs from this public repository
Changes to the docs can only be done through this repository

Is that what you're asking?

Reply:
![image](https://user-images.githubusercontent.com/15165028/149543806-1ef74ed2-b825-4b04-9385-6737f7693b7e.png)
Someone with the ability to add a webhook to this GitHub project will need to follow the guide linked here. First, they need to be added to the readthedocs project.